src/token/getTokens.ts

```diff
@@ -0,0 +1,57 @@
+import { ListSwapAssets } from '../definitions/swap';
+import { Token, RawTokenData, GetTokensOptions } from './types';
+import { sendRequest, JSONRPCError } from '../queries/request';
+
+/**
+ * Retrieves a list of tokens on Base.
+ * 
+ * @param {GetTokensOptions} options - An optional object that can be used to filter the list of tokens.
```

Comment:
Btw, to avoid duplication of docs, and kind of over work on writing them. We can avoid write the @parm here and just use the `site/docs/pages` as the only place where people should read the docs.

I think it's fair to still keep a description here, just for general context. But all the types live in the main doc site.